components/world_news_api/sources/news-event/news-event.mjs

```diff
@@ -0,0 +1,165 @@
+import app from "../../world_news_api.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New News",
+  description: "Emit new event when a new news is fetched. Calling this endpoint requires 1 point per page, up to 1000 news.",
```

Comment:
Did you take a look at [this suggestion](https://pipedream-users.slack.com/archives/G01J50YGJGP/p1680633796539479?thread_ts=1680617299.849229&cid=G01J50YGJGP)?

Reply:
Oh. I changed this, don't why is not here haha.